wombase_backend/server/apps/tools/serializers.py

```diff
@@ -8,11 +8,12 @@ class ToolSerializer(serializers.ModelSerializer):
     class Meta:
         model = Tool
         fields = (
```

Comment:
You should have two serializers - one for ListCreate with id and other for Details without id (like that which you have right now)